src/Plot/RobustPlot.purs

```diff
@@ -54,35 +75,58 @@ plotEnclosures canvasSize bounds domainSegments evaluator = segmentEnclosures
     -}
   toCanvasEnclosure x =
     let
-      xMidPointValue = map boundsA $ evaluator.f xMidPoint
+      (Tuple canvasXLower canvasXUpper) = bimap toCanvasX toCanvasX $ boundsNumber x
+
+      (Tuple xLA xUA) = boundsA x
+
+      xMidPoint = centreA x
+
+      enclosureWidth = xUA - xLA
+
+      twoA = fromRationalPrec 50 two
+
+      evaluatorX = evaluator2 x
 
-      xGradGrad = map boundsA $ evaluator.f'' x
+      xMidPointValue = boundsA <$> (evaluator xMidPoint <#> (_.value))
```

Comment:
Should this be written as `boundsA <$> (_.value)  <$>  evaluator xMidPoint` to be consistent with the rest of the file

Reply:
Done, thank you.